pipeline-model-api/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/ast/ModelASTSingleArgument.java

```diff
@@ -14,15 +14,21 @@
  * @author Andrew Bayer
  */
 public final class ModelASTSingleArgument extends ModelASTArgumentList {
+
+    /**
+     * While not @Nonnull, if this field is null parsing or validation errors will occur before
```

Comment:
I'm not sure how the Javadoc parser interprets `@Nonnull` literally, might cause errors depending on how strict it is (I think it's ok as long as it doesn't start a line, so it isn't seen as an unrecognized 
 Javadoc tag, but I'm not sure). If Javadoc is generated during the CI build here, then it's fine, but if it is only generated during releases, it could cause a problem.